ingenialink/ethercat/network.py

```diff
@@ -1,21 +1,18 @@
 import os
-import time
+import sys
+import platform
+import subprocess
+
+try:
+    import importlib.resources as pkg_resources
+except ImportError:
+    import importlib_resources as pkg_resources
 
-from pysoem import Master, INIT_STATE, BOOT_STATE
 import ingenialogger
 
 from ingenialink.network import NET_PROT
-from ingenialink.constants import (
-    FILE_EXT_SFU,
-    FILE_EXT_LFU,
-    FORCE_BOOT_PASSWORD,
-    FORCE_COCO_BOOT_IDX,
-    FORCE_COCO_BOOT_SUBIDX,
-    FOE_WRITE_PASSWORD,
-    FOE_WRITE_TIMEOUT,
-)
-from ingenialink.enums.ethercat import EC_STATE
 from ingenialink.exceptions import ILFirmwareLoadError
+from ingenialink import bin as bin_module
```

Comment:
So I guess now it is not possible to use FOE if the library is installed directly from the source, right?

Reply:
Yes, FOE application is needed and it is not included in the repo. I will include some info in the README

Reply:
I just add it.